### DISCUSSION TITLE
Show only cy.log in UI e2e testing?

### DISCUSSION THREAD

**Discussion by AlejandroSanchez90** [cypress-io/cypress#30142] [Questions and Help] [answered]
i have search around and cooldnt find an answer to this, is there a way to limit debug console in cypress ui to only cy.log and errors ? i see a bunch of http request, and every debug comment from commands...

**Comment by jennifer-shehane** [comment, marked as answer]
You can Hide HTTP requests within settings in cypress open mode. This is usually one of the noisier parts of the command log. https://on.cypress.io/open-mode#Hide-HTTP-Requests
